linkml_runtime/utils/schemaview.py

```diff
@@ -792,8 +825,8 @@ def permissible_value_parent(
                 pv = enum.permissible_values[permissible_value]
                 if pv.is_a:
                     return [pv.is_a]
-        else:
-            return []
```

Comment:
there is an implicit "return None" missing here if the `pv.is_a` condition is not met